olp-cpp-sdk-core/src/utils/Dir.cpp

```diff
@@ -339,7 +349,8 @@ std::string Dir::TempDirectory() {
     // it should never reach this path unless a big change in iOS
     ret = "/private/tmp";
   } else {
-    if (ret[ret.size() - 1] == '/') ret.pop_back();
+    if (ret[ret.size() - 1] == '/')
+      ret.pop_back();
```

Comment:
do we need braces here?
` if (ret[ret.size() - 1] == '/') {
      ret.pop_back();
}`